Gopkg.toml

```diff
@@ -65,5 +65,33 @@
   name = "github.com/giantswarm/microkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/operatorkit"
+
+[[override]]
```

Comment:
These overrides enforce version of transitive dependencies to be consistent with the direct ones; without them `dep ensure -update` would either not complete or would pick wrong (typically master) version/branch.